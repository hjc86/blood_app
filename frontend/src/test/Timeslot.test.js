// import React from 'react';
// import ReactDOM from 'react-dom'
// import {shallow} from 'enzyme';
// import renderer from 'react-test-renderer';
// import TimeslotCard from '../components/timeslots'

// describe('<TimeslotCard/>', () => {
    
// //   it('mounts to DOM', () => {
// //       const div = document.createElement('div');
// //       ReactDOM.render(<TimeslotCard />, div);
// //       ReactDOM.unmountComponentAtNode(div);
// //   });

//   let wrapper;
//   beforeEach(() => wrapper = shallow(<TimeslotCard />));
//   it('Renders correctly', () => {
//       expect(wrapper).toMatchSnapshot();
//   })

//   it('should render a <div />', () => {
//       expect(wrapper.find('div').length).toEqual(1);
//   });


// })