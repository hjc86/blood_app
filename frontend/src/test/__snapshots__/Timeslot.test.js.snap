// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeslotCard/> Renders correctly 1`] = `undefined`;
